Install hustle failed: cannot find arrow library

When installing hustle on macOS with the following procedure (in README.md):
1. Installed `brew`
2. `./install_requirements.sh ` and `./install_arrow.sh`. After the step, I see `hustle/arrow` directory.
3. `mkdir build && cd build`
4. `cmake -D CMAKE_BUILD_TYPE=RELEASE .. `. 

This failes with the following log:
```
-- Could NOT find PkgConfig (missing: PKG_CONFIG_EXECUTABLE) 
CMake Error at src/FindArrow.cmake:122 (message):
  Could not find the Arrow library.  Looked for headers in , and for libs in
Call Stack (most recent call first):
  src/CMakeLists.txt:21 (find_package)
```


**Question**
1. `cmake -v` on my machine is `3.18.2` instead of `3.15`. I think we didn't specify the cmake version in `Brewfile`. Shall we stick to that cmake version, or whatever an upper version is okay?
2. Am I missing a flag for header / lib path for cmake to search for arrow?



<details>
<summary>Full log</summary>

```
/Applications/CLion.app/Contents/bin/cmake/mac/bin/cmake -DCMAKE_BUILD_TYPE=Debug -G "CodeBlocks - Unix Makefiles" /Users/ginda/Documents/GitHub/hustle
-- The C compiler identification is AppleClang 11.0.3.11030032
-- The CXX compiler identification is AppleClang 11.0.3.11030032
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc - works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ - works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE  
-- Found PythonInterp: /usr/bin/python (found version "2.7.16") 
-- Looking for C++ include unistd.h
-- Looking for C++ include unistd.h - found
-- Looking for C++ include stdint.h
-- Looking for C++ include stdint.h - found
-- Looking for C++ include inttypes.h
-- Looking for C++ include inttypes.h - found
-- Looking for C++ include sys/types.h
-- Looking for C++ include sys/types.h - found
-- Looking for C++ include sys/stat.h
-- Looking for C++ include sys/stat.h - found
-- Looking for C++ include fnmatch.h
-- Looking for C++ include fnmatch.h - found
-- Looking for C++ include stddef.h
-- Looking for C++ include stddef.h - found
-- Check size of uint32_t
-- Check size of uint32_t - done
-- Looking for strtoll
-- Looking for strtoll - found
-- Check size of pthread_rwlock_t
-- Check size of pthread_rwlock_t - done
-- Performing Test COMPILER_HAS_WNO_UNUSED_LOCAL_TYPEDEF
-- Performing Test COMPILER_HAS_WNO_UNUSED_LOCAL_TYPEDEF - Success
-- Looking for C++ include dlfcn.h
-- Looking for C++ include dlfcn.h - found
-- Looking for C++ include execinfo.h
-- Looking for C++ include execinfo.h - found
-- Looking for C++ include glob.h
-- Looking for C++ include glob.h - found
-- Looking for C++ include libunwind.h
-- Looking for C++ include libunwind.h - found
-- Looking for C++ include memory.h
-- Looking for C++ include memory.h - found
-- Looking for C++ include pwd.h
-- Looking for C++ include pwd.h - found
-- Looking for C++ include stdlib.h
-- Looking for C++ include stdlib.h - found
-- Looking for C++ include strings.h
-- Looking for C++ include strings.h - found
-- Looking for C++ include string.h
-- Looking for C++ include string.h - found
-- Looking for C++ include syscall.h
-- Looking for C++ include syscall.h - not found
-- Looking for C++ include syslog.h
-- Looking for C++ include syslog.h - found
-- Looking for C++ include sys/syscall.h
-- Looking for C++ include sys/syscall.h - found
-- Looking for C++ include sys/time.h
-- Looking for C++ include sys/time.h - found
-- Looking for C++ include sys/ucontext.h
-- Looking for C++ include sys/ucontext.h - found
-- Looking for C++ include sys/utsname.h
-- Looking for C++ include sys/utsname.h - found
-- Looking for C++ include ucontext.h
-- Looking for C++ include ucontext.h - not found
-- Looking for C++ include unwind.h
-- Looking for C++ include unwind.h - found
-- Looking for fcntl
-- Looking for fcntl - found
-- Looking for sigalstack
-- Looking for sigalstack - not found
-- Check size of void*
-- Check size of void* - done
-- Performing Test HAVE___BUILTIN_EXPECT
-- Performing Test HAVE___BUILTIN_EXPECT - Success
-- Performing Test HAVE___ATTRIBUTE__
-- Performing Test HAVE___ATTRIBUTE__ - Success
-- Performing Test HAVE___SYNC_VAL_COMPARE_AND_SWAP
-- Performing Test HAVE___SYNC_VAL_COMPARE_AND_SWAP - Success
-- Checking program counter fetch from ucontext_t member: uc_mcontext.gregs[REG_EIP]
-- Performing Test PC_FROM_UCONTEXT_COMPILES
-- Performing Test PC_FROM_UCONTEXT_COMPILES - Failed
-- Checking program counter fetch from ucontext_t member: uc_mcontext.gregs[REG_RIP]
-- Performing Test PC_FROM_UCONTEXT_COMPILES
-- Performing Test PC_FROM_UCONTEXT_COMPILES - Failed
-- Checking program counter fetch from ucontext_t member: uc_mcontext.sc_ip
-- Performing Test PC_FROM_UCONTEXT_COMPILES
-- Performing Test PC_FROM_UCONTEXT_COMPILES - Failed
-- Checking program counter fetch from ucontext_t member: uc_mcontext.uc_regs->gregs[PT_NIP]
-- Performing Test PC_FROM_UCONTEXT_COMPILES
-- Performing Test PC_FROM_UCONTEXT_COMPILES - Failed
-- Checking program counter fetch from ucontext_t member: uc_mcontext.gregs[R15]
-- Performing Test PC_FROM_UCONTEXT_COMPILES
-- Performing Test PC_FROM_UCONTEXT_COMPILES - Failed
-- Checking program counter fetch from ucontext_t member: uc_mcontext.arm_pc
-- Performing Test PC_FROM_UCONTEXT_COMPILES
-- Performing Test PC_FROM_UCONTEXT_COMPILES - Failed
-- Checking program counter fetch from ucontext_t member: uc_mcontext.mc_eip
-- Performing Test PC_FROM_UCONTEXT_COMPILES
-- Performing Test PC_FROM_UCONTEXT_COMPILES - Failed
-- Checking program counter fetch from ucontext_t member: uc_mcontext.mc_rip
-- Performing Test PC_FROM_UCONTEXT_COMPILES
-- Performing Test PC_FROM_UCONTEXT_COMPILES - Failed
-- Checking program counter fetch from ucontext_t member: uc_mcontext.__gregs[_REG_EIP]
-- Performing Test PC_FROM_UCONTEXT_COMPILES
-- Performing Test PC_FROM_UCONTEXT_COMPILES - Failed
-- Checking program counter fetch from ucontext_t member: uc_mcontext.__gregs[_REG_RIP]
-- Performing Test PC_FROM_UCONTEXT_COMPILES
-- Performing Test PC_FROM_UCONTEXT_COMPILES - Failed
-- Checking program counter fetch from ucontext_t member: uc_mcontext->ss.eip
-- Performing Test PC_FROM_UCONTEXT_COMPILES
-- Performing Test PC_FROM_UCONTEXT_COMPILES - Failed
-- Checking program counter fetch from ucontext_t member: uc_mcontext->__ss.__eip
-- Performing Test PC_FROM_UCONTEXT_COMPILES
-- Performing Test PC_FROM_UCONTEXT_COMPILES - Failed
-- Checking program counter fetch from ucontext_t member: uc_mcontext->ss.rip
-- Performing Test PC_FROM_UCONTEXT_COMPILES
-- Performing Test PC_FROM_UCONTEXT_COMPILES - Failed
-- Checking program counter fetch from ucontext_t member: uc_mcontext->__ss.__rip
-- Performing Test PC_FROM_UCONTEXT_COMPILES
-- Performing Test PC_FROM_UCONTEXT_COMPILES - Success
-- Found program counter field in ucontext_t: uc_mcontext->__ss.__rip
CMake Warning (dev) at /Applications/CLion.app/Contents/bin/cmake/mac/share/cmake-3.17/Modules/FindPackageHandleStandardArgs.cmake:272 (message):
  The package name passed to `find_package_handle_standard_args` (PkgConfig)
  does not match the name of the calling package (Arrow).  This can lead to
  problems in calling code that expects `find_package` result variables
  (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  /Applications/CLion.app/Contents/bin/cmake/mac/share/cmake-3.17/Modules/FindPkgConfig.cmake:45 (find_package_handle_standard_args)
  src/FindArrow.cmake:10 (include)
  src/CMakeLists.txt:21 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Could NOT find PkgConfig (missing: PKG_CONFIG_EXECUTABLE) 
CMake Error at src/FindArrow.cmake:122 (message):
  Could not find the Arrow library.  Looked for headers in , and for libs in
Call Stack (most recent call first):
  src/CMakeLists.txt:21 (find_package)


-- Configuring incomplete, errors occurred!
See also "/Users/ginda/Documents/GitHub/hustle/cmake-build-debug/CMakeFiles/CMakeOutput.log".
See also "/Users/ginda/Documents/GitHub/hustle/cmake-build-debug/CMakeFiles/CMakeError.log".

[Failed to reload]
```
</details>

--- a/Brewfile
+++ b/Brewfile
@@ -1,1 +1,2 @@
 brew "cmake"
+brew "pkgconfig"

